app_demo.rb

```diff
@@ -0,0 +1,83 @@
+require './lib/order'
+require './lib/submitorder'
+
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1, :total, :to_mobile, :from_mobile, :message_body, :message, :account_sid, :auth_token, :submitorder1, :quantity
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+  def select_option
+    puts 'Takeaway Challenge'
+    puts '=================='
+    puts
+    puts 'Select Option:'
+    puts 'A - Add dish to menu'
+    puts 'B - Display menu so that can make an order'
+    puts 'C - Create an order'
+    puts 'D - Total an order'
+    puts 'E - Submit order'
+    puts 'X - End'
+    option = gets.chomp.downcase
+  end
+
+  def demo
+    begin
+      case select_option
+      when 'a'
+        while true
```

Comment:
Style/InfiniteLoop: Use `Kernel#loop` for infinite loops. (https://github.com/bbatsov/ruby-style-guide#infinite-loop)